base/Dockerfile

```diff
@@ -7,6 +7,6 @@ RUN apk -U upgrade && apk add --no-cache \
     ansible \
     py3-pip
 
-RUN pip install ansible-runner
+RUN pip install ansible-runner --break-system-packages
```

Comment:
unfortunately `apk add py3-ansible-runner` doesn't work, apparently it doesn't exist...

Reply:
Maybe it's because `py3-ansible-runner` is now installed with the apk `ansible` pkg ? In that case we should not reinstall it using pip and remove this whole line if it's already installed through the system packager.

Reply:
good point, let me test this theory :D

Reply:
nope, unfortunately it's not included :/

Reply:
Does not works 😞
Anyway, I think we should not mix pip and apk installs here, maybe the fix here is to install both ansible and ansible-runner through pip

Reply:
Let's also compare versions before and after to be sure we are not shipping BC breaks by the same time 😅

Reply:
Ha fuck no, the base image uses alpine 3.19, so ansible 8.6 https://pkgs.alpinelinux.org/packages?name=ansible&branch=v3.19&repo=&arch=&maintainer=

Reply:
So we should install ansible 8.6 from pip